### PR TITLE
Introduce doctoc to make Table of Contents update easily

### PR DESCRIPTION
**Pull request by PeterDaveHello**
So that there won't be inconsistent Table of Contents and it's very easy to update it - `npm run doctoc`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,6 +28,7 @@
     "boxen": "^1.0.0",
     "chalk": "^1.1.1",
     "default-pager": "^1.1.0",
+    "doctoc": "^1.3.0",
```
Comment (denysdovhan) [inline package.json]: Let's keep `doctoc` as dev dependency.

Comment (PeterDaveHello) [inline package.json]: Okay, done!

**Comment by denysdovhan** [comment]
Would be great if we find a way to do this automatically.

**Comment by PeterDaveHello** [comment]
It's possible! You can use deploy key to let Travis CI do that.

**Comment by PeterDaveHello** [comment]
Not immediately and I think it'll need write permission to the repo.

**Comment by denysdovhan** [comment]
Please, try to describe a process. I use the similar approach in my personal websites. If it's needed, I can encode an access-token for `.travis.yml` by myself.

**Pull request merged**